### DISCUSSION TITLE
Add android screen mirroring with scrcpy

### DISCUSSION THREAD

**Discussion by SuperninjaXII** [linuxmint/discussions#693] [Ideas]
To make Linux mint a great tool adding screen mirroring with an easy to use interface using scrypy and maybe way droid support will make it awesome

**Comment by Jopp-gh** [comment]
Agreed, to mirror Android on Mint would be very useful. The issue is, scrypy or how that software is called (what a horrendous name for an app) is outdated and Waydroid doesn't run smoothly.
 
Maybe somebody wants to create a new project with a decent performance.

Fun fact, you get all the tools for Android development on Linux, but almost zero support when you'd like to combine/mirror Android on Linux. Maybe a good, maybe a bad thing.

**Reply by SuperninjaXII** [reply]
I see , I didn't know.
I use scrypy and it works but am guessing it has to do with development activities on the repo

**Comment by SuperninjaXII** [comment]
https://github.com/Genymobile/scrcpy/releases/tag/v3.0

This released last week?

**Reply by Jopp-gh** [reply]
Oh cool, an update. 

Didn't know, let's see how it performs 🙂
